From fad4c140478243c9ceb4b8c1ed46c50e124f2b30 Mon Sep 17 00:00:00 2001
From: Xianzhong <xianzhong.li@nxp.com>
Date: Fri, 4 May 2018 19:19:39 +0800
Subject: [PATCH 3652/5242] MGS-3856-2 [#imx-1018] disable openvg2d for arm64
 build

commit  c0b00c2ff14a1a33e1ffba025b32284998e78b1e from
https://source.codeaurora.org/external/imx/linux-imx.git

MX8 chips does not have GC355. So no need to build the drivers

Signed-off-by : Xianzhong Li <xianzhong.li@nxp.com>
Reviewed-by : Prabhu Sundararaj <prabhu.sundararaj@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/mxc/gpu-viv/Kbuild |    4 ++++
 1 file changed, 4 insertions(+)

diff --git a/drivers/mxc/gpu-viv/Kbuild b/drivers/mxc/gpu-viv/Kbuild
index 39141de..a011a78 100644
--- a/drivers/mxc/gpu-viv/Kbuild
+++ b/drivers/mxc/gpu-viv/Kbuild
@@ -125,6 +125,10 @@ ifeq ($(VIVANTE_ENABLE_3D), 1)
 OBJS += $(ARCH_KERNEL_DIR)/gc_hal_kernel_recorder.o
 endif
 
+ifneq ($(CONFIG_ARM64),)
+VIVANTE_ENABLE_VG=0
+endif
+
 ifeq ($(VIVANTE_ENABLE_VG), 1)
 OBJS +=\
           $(HAL_KERNEL_DIR)/gc_hal_kernel_vg.o\
-- 
1.7.9.5

